base/bucket_gocb.go

```diff
@@ -1490,7 +1507,7 @@ func (bucket *CouchbaseBucketGoCB) deleteDocXattrOnly(k string, xattrKey string,
 	//  Do get w/ xattr in order to get cas
 	var retrievedVal map[string]interface{}
 	var retrievedXattr map[string]interface{}
-	getCas, err := bucket.GetWithXattr(k, xattrKey, &retrievedVal, &retrievedXattr)
+	getCas, err := bucket.GetWithXattr(k, xattrKey, "", &retrievedVal, &retrievedXattr, nil)
```

Comment:
This looks like technical debt that could get cleaned up, do you agree?  The only caller of deleteDocXattrOnly is deleteWithXattrInternal, which has already attempted to do a MutateInEx, which would have returned a cas.  It seems like that cas is the one we should be using here, and not doing a new Get.  If you agree, can you file a tracking ticket to clean this up?

Reply:
Agreed. Will file ticket, this whole 'area' regarding the delete can probably be cleaned up.

Reply:
CBG-1338